src/UserProfile/UserProfile.module.css

```diff
@@ -0,0 +1,235 @@
+.profilePage {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 100%;
+  max-width: 935px;
+  margin: 90px 30%;
+}
+.header {
+  display: flex;
+  border-bottom: 1px solid #dbdbdb;
+}
+.imgContainer {
+  height: 208px;
+  width: 291px;
+  margin-right: 30px;
+  display: flex;
+  justify-content: center;
+}
+.userInfoAndButtons {
+  display: flex;
+}
+.userName {
+  margin: 0;
+  display: block;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  font-weight: 300;
+  font-size: 28px;
+  line-height: 32px;
+  height: 45px;
+}
+.userImg {
+  width: 150px;
+  height: 150px;
+  border-radius: 50%;
+}
+.buttonsContainer {
+  display: flex;
+  gap: 8px;
+  height: 30px;
+  margin-left: 20px;
+}
+.messageButton,
+.followButton,
+.showMore {
+  height: 28px;
+  background-color: transparent;
+  border: 1px solid #dbdbdb;
+  height: 100%;
+  border-radius: 4px;
+  cursor: pointer;
+}
+.showMore {
+  width: 34px;
+}
+.DownIcon {
+  transform: rotate(180deg);
+}
+.threeDotsButton {
+  border: none;
+  background: none;
+  height: 28px;
+  cursor: pointer;
+}
+.followButton {
+  padding: 2px 13px;
+  box-sizing: border-box;
+  width: 74px;
+  color: rgba(26, 26, 26, 0.8);
+}
+.messageButton {
+  width: 75px;
+}
+.checkIcon {
+  transform: scale(0.7);
+  padding: 0;
+  margin-left: -7px;
+  color: rgba(26, 26, 26, 0.5);
+}
+.postInfo {
+  display: flex;
+  gap: 40px;
+}
+.count {
+  font-size: 16px;
+  cursor: pointer;
+}
+.postCount {
+  font-size: 16px;
+}
+.userInfo {
+  margin: 0;
+}
+.userInfoAndBio {
+  font-size: 16px;
+}
+.postVideosTagged {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  gap: 60px;
+}
+.link {
+  display: flex;
+  align-items: center;
+  gap: 5px;
+  text-decoration: none;
+  color: #8e8e8e;
+  height: 52px;
+}
+.isClicked {
+  color: #262626;
+  border-top: 1px solid #262626;
+}
+
+.postImg {
+  width: 293px;
+  height: 293px;
+  position: relative;
+  display: block;
+}
+
+.postsSectionPost {
+  display: flex;
+}
+.postsSectionPost:hover .postImg {
+  position: static;
+}
```

Comment:
I'm not sure, but I'm not sure you need this.

Reply:
You were right, I don't.